app/src/main/java/org/efidroid/efidroidmanager/types/FSTabEntry.java

```diff
@@ -105,6 +105,26 @@ public boolean isMultiboot() {
         return false;
     }
 
+    public boolean isLokiAboot() {
```

Comment:
the name should be 'isLokiABoot' because A is for Android and B is for Bootloader.